eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/Primitives/KnownDecorators.cs

```diff
@@ -5,16 +5,12 @@ namespace Azure.Generator.Management.Primitives
 {
     internal class KnownDecorators
     {
-        public const string ArmResourceOperations = "Azure.ResourceManager.@armResourceOperations";
-        public const string ArmResourceRead = "Azure.ResourceManager.@armResourceRead";
-        public const string ArmProviderNamespace = "Azure.ResourceManager.@armProviderNamespace";
         public const string ResourceMetadata = "Azure.ClientGenerator.Core.@resourceSchema";
-        public const string ArmResourceInternal = "Azure.ResourceManager.Private.@armResourceInternal";
 
-        public const string ResourceClient = "resourceClient";
-        public const string ResourceModel = "resourceModel";
         public const string ResourceType = "resourceType";
         public const string IsSingleton = "isSingleton";
         public const string ResourceScope = "resourceScope";
+        public const string Methods = "methods";
+        public const string ParentResource = "parentResource";
```

Comment:
I think the only place that we would use these constant is the `ManagementInputLibrary` class.
Maybe we could simply this by defining these constant right in the  `ManagementInputLibrary` class instead and remove this type.